src/bundle/Resources/public/js/scripts/helpers/cookies.helper.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc, eZ) {
+    const setCookie = (name, value, maxAgeDays = 356, path = '/') => {
+        const maxAge = maxAgeDays * 24 * 60 * 60;
+
+        doc.cookie = `${name}=${value};max-age=${maxAge};path=${path}`;
+    };
+    const getCookie = (name) => {
+        const cookieName = name + '=';
```

Comment:
nitpick: this is actually not a cookie name, but cookie name with equal sign.
Personally, because it is very short and straightforward, I would prefer to remove this and just add `` `${name}=` `` inside `cookieString.indexOf`.